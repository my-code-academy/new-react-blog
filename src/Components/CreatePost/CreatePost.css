.button{
    background-color:coral ;
    border: none;
    color: white;
    padding: 15px 32px;
    text-align: center;
    text-decoration: none;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;

  
}

.createPost{
    display:flex;
    justify-content: center;
}