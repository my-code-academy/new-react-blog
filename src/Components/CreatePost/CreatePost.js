import React, { Component } from 'react';
import './CreatePost.css';

class CreatePost extends Component {
  render() {
    return (
        <div className="createPost">
        <button className="button">CreatePost</button>
            

        </div>
    );
  }
}

export default CreatePost;
