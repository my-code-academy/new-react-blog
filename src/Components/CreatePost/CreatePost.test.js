import React from 'react';
import renderer from 'react-test-renderer';
import CreatePost from './CreatePost';

describe('CreatePost', () => {
  it("should render a CreatePost button", () => {
    const tree = renderer.create(<CreatePost/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

