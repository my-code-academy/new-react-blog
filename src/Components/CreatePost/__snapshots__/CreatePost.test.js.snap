// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreatePost should render a CreatePost button 1`] = `
<div
  className="createPost"
>
  <button
    className="button"
  >
    CreatePost
  </button>
</div>
`;
